README.md

```diff
@@ -79,6 +79,67 @@ See more about chart settings [here](https://github.com/grafana/helm-charts/blob
 
 Another option would be to build custom Grafana image with plugin based on same installation instructions.
 
+## Plugin provision
+
+Provision of grafana plugin requires to:
```

Comment:
I'd expect to have something like this https://github.com/Altinity/clickhouse-grafana#configure-the-datasource-with-provisioning

1. Reference to grafana docs
2. Fields which are important for our plugin, like type

All the extra steps with docker-compose are nice, but not always a case for users.

Reply:
I checked different data source plugins; many still need examples of creating or checking their provisioning config. So I have updated the doc. Added more clear example of the provision config file and described the steps on how to check the configuration